<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="action_ReadShp"/>
     <addaction name="action_3"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Edit_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Draw"/>
   <addaction name="action_Move"/>
   <addaction name="action_Edit"/>
   <addaction name="separator"/>
   <addaction name="action_Save"/>
   <addaction name="action_Refresh"/>
   <addaction name="separator"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="action_movescene"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Tcp_Server"/>
   <addaction name="action_Tcp_Connect"/>
   <addaction name="action_Tcp_Sent"/>
   <addaction name="action_Tcp_Time"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>350</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>画图工具</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择图形：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="shapeComboBox">
         <property name="currentText">
          <string>点</string>
         </property>
         <property name="maxCount">
          <number>2147483642</number>
         </property>
         <item>
          <property name="text">
           <string>点</string>
          </property>
          <property name="icon">
           <iconset resource="ICON/icon.qrc">
            <normaloff>:/new/prefix1/circle.png</normaloff>:/new/prefix1/circle.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>线</string>
          </property>
          <property name="icon">
           <iconset resource="ICON/icon.qrc">
            <normaloff>:/new/prefix1/polyline.png</normaloff>:/new/prefix1/polyline.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>面</string>
          </property>
          <property name="icon">
           <iconset resource="ICON/icon.qrc">
            <normaloff>:/new/prefix1/polygon.png</normaloff>:/new/prefix1/polygon.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>画笔类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="penStyleComboBox">
         <item>
          <property name="text">
           <string>实线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>虚线</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>画笔线宽：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="penWidthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>12</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>画笔颜色：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBrowser_Pen">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="penColorToolButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>11</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>填充颜色：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBrowser_Brush">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="brushColorToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
    <zorder>verticalSpacer</zorder>
    <zorder>verticalSpacer_2</zorder>
    <zorder>verticalSpacer_3</zorder>
    <zorder>verticalSpacer_4</zorder>
    <zorder>verticalSpacer_5</zorder>
    <zorder>label_6</zorder>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Layers">
   <property name="minimumSize">
    <size>
     <width>109</width>
     <height>134</height>
    </size>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Move">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/move.png</normaloff>:/new/prefix1/move.png</iconset>
   </property>
   <property name="text">
    <string>移动要素(&amp;M)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/edit.png</normaloff>:/new/prefix1/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑要素(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/save.ico</normaloff>:/new/prefix1/save.ico</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/zoom_in.ico</normaloff>:/new/prefix1/zoom_in.ico</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Shift+Up</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/zoom_out.ico</normaloff>:/new/prefix1/zoom_out.ico</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Shift+Down</string>
   </property>
  </action>
  <action name="action_Rotate">
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="action_Refresh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/delete.png</normaloff>:/new/prefix1/delete.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="action_Draw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/NewShape.ico</normaloff>:/new/prefix1/NewShape.ico</iconset>
   </property>
   <property name="text">
    <string>绘制要素(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>绘制要素(D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_ReadShp">
   <property name="text">
    <string>从文件</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>从数据库</string>
   </property>
  </action>
  <action name="action_Tcp_Time">
   <property name="text">
    <string>启动定时</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;10秒自动同步一次&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_Tcp_Connect">
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;连接服务端&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_Tcp_Sent">
   <property name="text">
    <string>发送数据</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;同步服务器数据&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_Edit_2">
   <property name="text">
    <string>开始编辑</string>
   </property>
  </action>
  <action name="action_Tcp_Server">
   <property name="text">
    <string>启动</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;启动服务端&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="action_movescene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ICON/icon.qrc">
     <normaloff>:/new/prefix1/pan.png</normaloff>:/new/prefix1/pan.png</iconset>
   </property>
   <property name="text">
    <string>移动地图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ICON/icon.qrc"/>
 </resources>
 <connections/>
</ui>
